fritzconnection/lib/fritzhosts.py

```diff
@@ -18,6 +18,17 @@
 SERVICE = "Hosts1"
 
 
+def _get_mesh_by_mac(topology, mac):
+    if topology is None:
+        return None, None
+
+    for node in topology["nodes"]:
+        if node["device_mac_address"] == mac:
+            return node["is_meshed"], node["mesh_role"]
```

Comment:
Maybe we can 

1. force is_meshed to be bool
2. change `mesh_role` from `unknown` or `None` to become `"None"` (str)

Reply:
Thanks for your feedback. Tried to address it in latest commit :D